Buddies/Activities/ActivityTableVC.swift

```diff
@@ -7,84 +7,125 @@
 //
 
 import UIKit
+import Firebase
 
 class ActivityTableVC: UITableViewController {
-
+    
+    var stopListeningToUser: Canceler?
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        self.tableView.rowHeight = 100
         // Uncomment the following line to preserve selection between presentations
         // self.clearsSelectionOnViewWillAppear = false
-
+        
         // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
         // self.navigationItem.rightBarButtonItem = self.editButtonItem
     }
-
+    
     // MARK: - Table view data source
-
+    
     override func numberOfSections(in tableView: UITableView) -> Int {
         // #warning Incomplete implementation, return the number of sections
-        return 0
+        return 1
     }
-
+    
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         // #warning Incomplete implementation, return the number of rows
-        return 0
+        return 1
     }
-
-    /*
+    
+    //USES TEST ACTIVITY RN
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "reuseIdentifier", for: indexPath)
-
+        
+        let cell = tableView.dequeueReusableCell(withIdentifier: "activityCell", for: indexPath) as! ActivityCell
+        
+        self.stopListeningToUser = DataAccessor.instance.useActivity(id: "EgGiWaHiEKWYnaGW6cR3"){ activity in
```

Comment:
1. This shouldn't overwrite `stopListeningToUser`. This should be an array.
2. I think we should ask for activities we need in something called from `viewDidLoad`. It would be easier in my mind to store activities in a local `[ActivityId : Activity]` and then trigger a table row update on change.